codalab/worker/docker_image_manager.py

```diff
@@ -22,6 +22,7 @@
 ImageCacheEntry = namedtuple(
     'ImageCacheEntry', ['id', 'digest', 'last_used', 'virtual_size', 'marginal_size']
 )
+DOCKER_ERROR_SUBSTRING = "(must be forced)"
```

Comment:
Since we use this constant only once, I would actually not store this as a constant variable but directly compare against the string literal. That way the string is at the same place as the comments that explain what is going on. I'm afraid if it's like this this constant would be confusing to someone looking at this codebase for the first time.

Reply:
Yes. I feel this is quite hacky as well. I'll remove this and compare it directly with the error message.